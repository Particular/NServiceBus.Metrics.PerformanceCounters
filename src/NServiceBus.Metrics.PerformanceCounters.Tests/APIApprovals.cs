using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(PerformanceCountersFeature).Assembly);
        Approver.Verify(publicApi);
    }
}