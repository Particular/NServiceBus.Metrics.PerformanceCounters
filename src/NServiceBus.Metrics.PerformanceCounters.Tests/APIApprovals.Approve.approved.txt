namespace NServiceBus
{
    public class static PerformanceCountersExtensions
    {
        public static NServiceBus.PerformanceCountersSettings EnableWindowsPerformanceCounters(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
    }
    public class PerformanceCountersSettings
    {
        public void EnableSLAPerformanceCounters(System.TimeSpan sla) { }
    }
}